### PR TITLE
Ns template code

### PR DESCRIPTION
**Pull request by natibekele**
**Thymeleaf template files** 

- layout templates for languages, product, book and chapter download pages

- header, hero, navbar, searchbar, and multiple card fragments

- css files for styling of layouts/fragments

- additional assets

---
This change is [Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/25)

### CODE REVIEW

File: src/main/resources/templates/chapters.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <a class="chapter-download-card__button"
+        th:href="${'/'+ 'replace this with url from book object'}">
+        <em class="material-icons">get_app</em>
+        <p >Download Book</p>
+    </a>
+</div>
+
+<div class="l-list-container" >
+    <div th:each="language: ${languageList}">
+        <div th:insert="fragments/chaptercard :: chaptercard(${language.chapterNumber}, ${language.localizedName}, ${language.url}, '#')"></div>
```
Comment (AnonymousWalker) [inline src/main/resources/templates/chapters.html:33]: Did you forget to update the variable?

Comment (natibekele) [reply src/main/resources/templates/chapters.html:33]: oh yes, my fault. i was using that to test the template.

File: src/main/resources/templates/fragments/productcard.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    </div>
+    <div class="product-card__text"
+         th:classappend="${url == false}? 'unavailable' ">
+        <p th:text="${productTitle}"class="product-card__title"
+           th:classappend="${url == false}? 'unavailable' ">አማርኛ</p>
+        <p th:text="${productDescription}" class="product-card__description"
+           th:classappend="${url == false}? 'unavailable' "><span>Amharic</span></p>
+    </div>
+    <span class="product-card__icon"
+          th:classappend="${url == false}? 'unavailable' ">
```
Comment (AnonymousWalker) [inline src/main/resources/templates/fragments/productcard.html:22]: This also needs to be updated.

File: src/main/resources/templates/fragments/header.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+<head>
+    <title>Header</title>
+</head>
+<body>
+<header th:fragment="header" class="header">
+    <img alt="app_logo" class="header__logo" src="static/fetcher_logo.png" th:src="@{static/img/fetcher_logo.png}">
```
Comment (AnonymousWalker) [inline src/main/resources/templates/fragments/header.html:8]: Hey Nathan, this one may not need th:src

File: src/main/resources/templates/chapters.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <a class="chapter-download-card__button"
+        th:href="${'/'+ 'replace this with url from book object'}">
+        <em class="material-icons">get_app</em>
+        <p >Download Book</p>
+    </a>
+</div>
+
+<div class="l-list-container" >
+    <div th:each="chapter: ${chapterList}">
+        <div th:insert="fragments/chaptercard :: chaptercard(${chapter.chapterNumber}, ${chapter.localizedName}, ${chapter.url})"></div>
```
Comment (AnonymousWalker) [inline src/main/resources/templates/chapters.html:33]: chapter should have: 
    chapterNumber: Int,
    titleKey: String,
    url: String

**Pull request merged**